AutoLayoutHelper.xcodeproj/project.pbxproj

```diff
@@ -346,6 +347,7 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				INFOPLIST_FILE = AutoLayoutHelper/Info.plist;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
+				PRODUCT_BUNDLE_IDENTIFIER = "com.madhikarma.$(PRODUCT_NAME:rfc1034identifier)";
```

Comment:
No its not :) I ran a project settings update using Xcode 7.0 and it inserted this for me so i'm guessing its either  
- using a default organisation / company setting 
- this was something originally set in the project i.e. before it was forked to ustwo from my personal account 

![screen shot 2015-10-14 at 10 18 23](https://cloud.githubusercontent.com/assets/1456685/10479567/0f278a42-725d-11e5-8ed6-d4e6436fda58.png)